datalad/support/gitrepo.py

```diff
@@ -3337,7 +3337,16 @@ def save_(self, message=None, paths=None, _status=None, **kwargs):
                     # we would always see individual files
                     recursive=False):
                 # TODO normalize result
-                yield r
+                yield get_status_dict(
+                    action='delete',
+                    refds=self.pathobj,
+                    # TODO make remove() report the type
+                    # for now it claims to report on files only
```

Comment:
I would have  that assumption coded in, so whenever it fails we would know where to fix. Feels strange that original `r` is completely ignored (besides taking part in `path`).

Reply:
It is not clear to me what is requested.